modxcms/revolution
# [3.x] Fix resource tree overflow on Firefox (and fix scrollbar color for Firefox Linux)

### What does it do?
This PR makes two changes:

1. Fixes the resource tree overflow problem for Firefox as described in issue #15425 
2. Sets a default color for scrollbars in the 3.x manager theme when using Firefox on Linux as described in issue #12991 

Note that the `scrollbar-color` property used here is only supported by Firefox, so it won't have any bearing on other browsers.

### Why is it needed?
To create a more uniform look for the manager across different browsers/platforms.

### How to test
Compile the sass files and load the 3.x manager using Firefox. Look for a scrollbar on the resource tree, and take notice of whether scrollbars match the theme or not.
It would be great if people could test it on Windows/Mac as well and make sure there are no adverse side effects as I've only got Linux machines.

### Related issue(s)/PR(s)
#15425 , #12991
